rust/typeql_lang.rs

```diff
@@ -20,5 +20,70 @@
  *
  */
 
+#![allow(dead_code)]
```

Comment:
Shouldn't be present in the finished version, but useful for now so that the unimplemented (and therefore unused) visitors don't clutter the warnings.